### PR TITLE
feat(pie-assistive-text): DSW-1590 added boilerplate code using the generator

### PR DESCRIPTION
**Pull request by LTurns**
## Describe your changes (can list changeset entries if preferable)

[Added] - New pie-assistive-text component with boilerplate code

### CODE REVIEW

File: packages/components/pie-assistive-text/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "lint:scripts": "run -T eslint .",
+    "lint:scripts:fix": "yarn lint:scripts --fix",
+    "lint:style": "run -T stylelint ./src/**/*.{css,scss}",
+    "lint:style:fix": "yarn lint:style --fix",
+    "watch": "run -T vite build --watch",
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "test:ci": "yarn test",
+    "test:browsers": "npx playwright test -c ./playwright-lit.config.ts",
+    "test:browsers:ci": "yarn test:browsers",
+    "test:visual": "echo \"Error: A new Percy project needs to be created before visual tests can be run. steps: https://vue.pie.design/?path=/story/documentation-standards-testing-visual-regression--page \" && exit 1",
```
Comment (raoufswe) [inline packages/components/pie-assistive-text/package.json:27]: i think you need to update the test:visual command here to follow these [steps](https://www.figma.com/file/OOgnT2oNMdGFytj5AanYvt/%5BCore%5D-Web-Component-Documentation-%5BPIE-3%5D?node-id=1148%3A14397&mode=dev)

Comment (raoufswe) [reply packages/components/pie-assistive-text/package.json:27]: you will also need to update the project label configs as mentioned[ in the readme](https://github.com/justeattakeaway/pie/tree/main/packages/tools/generator-pie-component#setting-up-the-project-label-for-github)

Comment (LTurns) [reply packages/components/pie-assistive-text/package.json:27]: oh sorry didn't realise I had to do that in this PR, it's already added to percy so can do that now 👍

Comment (raoufswe) [reply packages/components/pie-assistive-text/package.json:27]: and lastly, u will need to add the component to storybook as mentioned [here](https://github.com/justeattakeaway/pie/tree/main/packages/tools/generator-pie-component#setting-up-storybook)

Comment (raoufswe) [reply packages/components/pie-assistive-text/package.json:27]: here is how a full example should look [like](https://github.com/justeattakeaway/pie/pull/1229), happy to hop on a call if it's easier 🤝

**Review by raoufswe — APPROVED** [review]

**Comment by jamieomaguire** [comment]
@LTurns please create a new label in Github for `pie-assistive-text` and add it to this PR before merging? :)

**Review by jamieomaguire — APPROVED** [review]

**Comment by raoufswe** [comment]
@jamieomaguire I believe Lizzie already added to `.github/project-labeler.yml` and that will only apply once we merge the PR.

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**